opensrp-reveal/src/main/java/org/smartregister/reveal/util/RevealJsonFormUtils.java

```diff
@@ -327,8 +330,6 @@ public String getFormName(String encounterType, String taskCode) {
         } else if (Constants.EventType.PAOT_EVENT.equals(encounterType) || Intervention.PAOT.equals(taskCode)) {
             if (BuildConfig.BUILD_COUNTRY == Country.THAILAND) {
                 formName = JsonForm.THAILAND_PAOT_FORM;
-            } else if (BuildConfig.BUILD_COUNTRY == Country.REFAPP) {
```

Comment:
I see a lot of form changes from the REFAPP form to either the NAMIMBIA or THAILAND version. Can you confirm that this is intentional?